### ISSUE TITLE
unable to convert R versions data frame to tibble

### ISSUE THREAD

**Issue by espinielli** [r-hub/rversions#20]
I tried to convert the data frame returned by `r_versions()` to a `tibble` but got an error message about the `date` column being of `POSIXlt` which is not supported by `dplyr` (see tidyverse/dplyr#179)

I found a workaround as per my repex below, but I was wondering what is the rationale to store the release date as `POSIXlt` (introduced as fix for r-hub/rversions#13) rather than `POSIXct` (which would make it ok for `dplyr`)...

Here is my rexex for this issue:

``` r
# remotes::install_github("r-hub/rversions")
library(rversions)
library(dplyr)
#> Warning: package 'dplyr' was built under R version 3.6.1
#> 
#> Attaching package: 'dplyr'
#> The following objects are masked from 'package:stats':
#> 
#>     filter, lag
#> The following objects are masked from 'package:base':
#> 
#>     intersect, setdiff, setequal, union

# fails
r_versions() %>% as_tibble()
#> Column `date` is a date/time and must be stored as POSIXct, not POSIXlt.

# fails too, for context see https://github.com/tidyverse/dplyr/issues/179 
r_versions() %>%
  mutate(date = as.character(date)) %>% 
  mutate(date = lubridate::as_datetime(date))
#> Column `date` is a date/time and must be stored as POSIXct, not POSIXlt.

# workaround: convert Posixlt column to char/datetime
vers <- r_versions()
vers$date <- as.character(vers$date) %>%
  lubridate::as_datetime()

# success
vers %>% as_tibble()
#> # A tibble: 114 x 3
#>    version date                nickname
#>    <chr>   <dttm>              <chr>   
#>  1 0.60    1997-12-04 08:47:58 <NA>    
#>  2 0.61    1997-12-21 13:09:22 <NA>    
#>  3 0.61.1  1998-01-10 00:31:55 <NA>    
#>  4 0.61.2  1998-03-14 19:25:55 <NA>    
#>  5 0.61.3  1998-05-02 07:58:17 <NA>    
#>  6 0.62    1998-06-14 12:56:20 <NA>    
#>  7 0.62.1  1998-06-14 22:13:25 <NA>    
#>  8 0.62.2  1998-07-10 11:13:45 <NA>    
#>  9 0.62.3  1998-08-28 09:02:19 <NA>    
#> 10 0.62.4  1998-10-23 12:08:41 <NA>    
#> # ... with 104 more rows
```

<sup>Created on 2019-08-22 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by maelle** [comment]
thanks, fixed in #21

**Comment by maelle** [comment]
@espinielli the fix is on CRAN now.
